Ordering/OrderManagement/OrderFramework.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Ordering.Common.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Ordering.BusinessLogic.OrderManagement
+{
+    class OrderFramework:IOrderManager
+    {
+        public SortedList<Object, int> newOrder = new SortedList<object, int>();
+
+
+        public void AddItem(Object Item, int Quantity)
```

Comment:
item?

Reply:
Oswald is writing the code for order item separately, and was going to submit his own pull request for it. Would you rather we combine it first, and submit it all at once?